### DISCUSSION TITLE
How can I report issue about composition of dependencies?

### DISCUSSION THREAD

**Discussion by geniys** [microsoft/vcpkg#31933] [Q&A] [answered]
I found issue when I set the OpenSSL cipher at specific composition.
Where is right place to report it?

Simple description of issue is:
Not understandable error occurred when I call SSL_CTX_set_cipher_list method.
It occurred on Ubuntu 22.04 (x64) with latest apache thrift and OpenSSL 1.1.1n#1.
It doesn't reproduced on Windows or apple silicon.

**Comment by autoantwort** [comment, marked as answer]
Is this a vcpkg specific issue? If not, report this to the openssl maintainers or ask on stackoverflow
